app/components/footer.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <footer className="bg-gray-100 text-black py-4 border-t">
```

Comment:
Question to styling: shouldn't the footer be always in the end of a page? Especially when there is not enough content on a page
 
<img width="933" alt="image" src="https://github.com/user-attachments/assets/f9fc2088-3d97-4969-b04a-61781236e367" />